extensions/default/src/DicomJSONDataSource/index.js

```diff
@@ -11,6 +11,7 @@ const mappings = {
   patientId: 'PatientID',
 };
 
+let _data;
```

Comment:
can we instead augment our _store to include the studyInstanceUIDs for each URL (I thought we do) instead of adding a new cache?